src/data/markdown/docs/30 xk6-browser/01 xk6-browser/03 BrowserType.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "BrowserType"
+excerpt: "xk6-browser: BrowserType Class"
+---
+
+The `BrowserType` is the entry point into launching a browser process; `chromium` is currently the only supported `BrowserType`. To use it, import `chromium` from the top level module `/k6/x/browser`.
+
+| Method            | Description                                                                                                      |
+|-------------------|------------------------------------------------------------------------------------------------------------------|
+| launch([options]) | Launches a new browser process. You can see a list of all possible `options` in [the options heading](#options). |
+
+### Options
```

Comment:
To clarify:
```suggestion
### `launch()` options
```

We'll have to change it anyway once we implement `connect()` and `launchPersistentContext()`, which also take an `options` object.

Reply:
I've made the larger change so that the options for `launch` now live in its own page. I've not created pages for `connect` or `laucnPersistentContext` since they're not implemented but documented them as WIP.

Resolved in: 805dd25465b382f0e5edc1797adad4ac3000d8a1